### PR TITLE
Prep release 1.1 branch for release candidate 

### PR DESCRIPTION
**Pull request by m-m-adams**
- Disable the freeze on dangling cluster pointers in non-beta builds, it's a small memory leak and should get cleared at song change now anyway
- Turn off alpha/beta version
- Set build type to release
- pull out midi loopback due to the weird playhead issues

### CODE REVIEW

File: src/deluge/model/sample/sample.cpp [hunk 1]
```diff
@@ -1794,7 +1794,10 @@ void Sample::numReasonsDecreasedToZero(char const* errorCode) {
 
 		// LegsMechanical got, V4.0.0-beta2.
 		// https://forums.synthstrom.com/discussion/4106/v4-0-beta2-e078-crash-when-recording-audio-clip
+#if ALPHA_OR_BETA_VERSION
+		// this occurs very rarely and should only leak one cluster - better to not freeze in the official build
 		FREEZE_WITH_ERROR("E078");
+#endif
```
Comment (sapphire-arches) [inline src/deluge/model/sample/sample.cpp:1800]: this logic is already guarded

**Review by sapphire-arches — COMMENTED** [review]
If we merge this to 1.1 we'll lose a bunch of sanity checks but maybe that's fine and we can reenable them after the RC build?

File: src/deluge/model/sample/sample.cpp (hunk 1 above)
Comment (m-m-adams) [reply src/deluge/model/sample/sample.cpp:1800]: Oh you're right, I missed that the whole function was wrapped in it

**Comment by m-m-adams** [comment]
> If we merge this to 1.1 we'll lose a bunch of sanity checks but maybe that's fine and we can reenable them after the RC build?

Yeah, my thinking is that it's ok because we're only seeing E078 come up right now and we know in that case freezing is worse than leaking a small amount of memory. But yeah flipping it on again after making the RC is a good idea

**Review by sapphire-arches — APPROVED** [review]

**Pull request merged**